tests/e2e/driver/CheReporter.ts

```diff
@@ -175,8 +183,14 @@ class CheReporter extends mocha.reporters.Spec {
 
       // stop and remove running workspace
       if (TestConstants.DELETE_WORKSPACE_ON_FAILED_TEST) {
-        console.log('Property DELETE_WORKSPACE_ON_FAILED_TEST se to true - trying to stop and delete running workspace.');
-        testWorkspaceUtil.cleanUpAllWorkspaces();
+        Logger.warn('Property DELETE_WORKSPACE_ON_FAILED_TEST se to true - trying to stop and delete running workspace.');
+        const workspaceID: string = await testWorkspaceUtil.getIdOfRunningWorkspace(CheReporter.latestWorkspace);
+        Logger.debug(`Stopping workspace:${CheReporter.latestWorkspace} with ID:${workspaceID}`);
+        testWorkspaceUtil.stopWorkspaceById(workspaceID);
+        Logger.debug(`Waiting for workspace to switch to stopped state.`);
+        testWorkspaceUtil.waitWorkspaceStatus(`${TestConstants.TS_SELENIUM_USERNAME}-che`, CheReporter.latestWorkspace, WorkspaceStatus.STOPPED);
+        Logger.debug(`Deleting workspace {${workspaceID}}`);
+        testWorkspaceUtil.removeWorkspaceById(workspaceID);
```

Comment:
What do you think about moving this logic to separate method in testWorksapceUtil? Something similar to `testWorkspaceUtil.cleanUpAllWorksapces()`, e.g. `testWorksapceutil.cleanWorkspace`?

Reply:
refactored.